awx/api/templates/swagger/description.md

```diff
@@ -0,0 +1 @@
+The Ansible Tower API Reference Manual provides in-depth documentation for Tower's REST API, including examples on how to integrate with it.
```

Comment:
This is a markdown the docs team can customize to provide a sort of intro paragraph or two at the top of the docs.

Reply:
@tvo318 @docschick ^